teleexception/__init__.py,sha256=Fz1y-Kx2hVMKRQR1U_DhbKpjdI1u24RK5DcKzNOqkbc,97
teleexception/status/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
teleexception/status/http_status.py,sha256=7uC8S6oehGLngwQzKmpe3W30XNFkQ5a42QpV8l6wiV8,13229
teleexception/status/status_exception.py,sha256=syVOIKg72y_Tjv_ndRuxIJ-ec_HGF3jDz_xqFj4WFQY,361
teleexception-0.1.9.dist-info/METADATA,sha256=hPmy0Z-XF_MbQKEBGA9U7dL9IDPp-Z5xtRHxH2Jw1rU,809
teleexception-0.1.9.dist-info/WHEEL,sha256=G16H4A3IeoQmnOrYV4ueZGKSjhipXx8zc8nu9FGlvMA,92
teleexception-0.1.9.dist-info/top_level.txt,sha256=8UE4OrNUAjQojJfld8TkzSD2RaYtO4XlXHzQIGBrTXU,14
teleexception-0.1.9.dist-info/RECORD,,
